src/react-apps/templates/template-app/src/store/index.ts

```diff
@@ -25,7 +25,7 @@ function configureStore(initialState?: any): Store<IServiceDevelopmentAppState>
     enhancer = compose(applyMiddleware(...middlewares));
   }
 
-  const createdStore: Store<IServiceDevelopmentAppState> = createStore(
+  const createdStore: Store<ITemplateAppState> = createStore(
```

Comment:
Rename from "state" to "store"? 
We do not want to confuse redux store with react component state.